### PR TITLE
Drop privileges

### PR DESCRIPTION
**Pull request by iyesin**
Drop privileges by default

### CODE REVIEW

**Comment by bobrik** [comment]
Why isn't `USER nobody` enough?

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (bobrik) [inline Dockerfile]: This seems unnecessary.

File: Dockerfile [hunk 2]
```diff
@@ -6,4 +7,6 @@ RUN apk --update add python py-setuptools py-pip && \
     apk del py-pip && \
     rm -rf /var/cache/apk/*
 
+USER nobody:nobody
```
Comment (bobrik) [inline Dockerfile:9]: Does the group have to be explicit here?

File: Dockerfile (hunk 1 above)
Comment (bobrik) [reply Dockerfile]: Please also squash after removing this.

**Comment by iyesin** [comment]
@bobrik you're right. That's should be enough.

**Review by bobrik — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (iyesin) [inline Dockerfile]: @bobrik already did it before your comment appeared.

File: Dockerfile (hunk 2 above)
Comment (iyesin) [reply Dockerfile:9]: @bobrik This is just an attempt to be safe from situations described in warning block here https://docs.docker.com/engine/reference/builder/#user

**Pull request merged**